modules/ROOT/edits/limitations-self.adoc

```diff
@@ -0,0 +1,19 @@
+= Limitations with Anypoint Runtime Fabric on Self-Managed Kubernetes
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+The following information is important when installing, configuring, and managing Anypoint Runtime Fabric on Self-Managed Kubernetes.
+
+[cols="2*a"]
+.Anypoint Runtime Fabric on Self-Managed Kubernetes resource allocation 
+|===
+| Nodes | The maximum number of nodes supported is 30.
+| Node Types | VM-based nodes are required. For example, Fargate is not supported.
+| Replicas Per Node | The maximum number of replicas that can be deployed per node is 40.
+
+Run no more than 20 - 25 replicas per core, up to a maximum of 40 replicas per node, to allow core sharing across replicas when needed for bursting. This ensures the Runtime Fabric’s internal components that run on each worker node are not overloaded by too many replicas.
```

Comment:
GLOBALLY avoid using " - " in indicate a span; instead, use either "to" or "through" or similar precise rewording/

Reply:
"Run no more than 20 - 25 replicas per core, up to a maximum of 40 replicas per node," is confusing: are we limiting them to between 20 and 25 replicas, or are we limiting them to 40? Perhaps...:
```suggestion
Although the maximum replicas per node is 40, it is best to run no more than between 20 and 25 replicas per core,  to allow core sharing across replicas when needed for bursting. This helps to ensure that Runtime Fabric internal components that run on each worker node are not overloaded by too many replicas.
```
Note that we must GLOBALLY avoid using "ensure" as a promise of functionality, for legal reason. Replace with "helps to ensure," per SR.